### DISCUSSION TITLE
/etc/conf.d/mac Not working with latest firmware but working in old firmware

### DISCUSSION THREAD

**Discussion by rafiomeon** [AxisCommunications/discussions#126] [ACAP] [answered]
Hi,

We have an Acap application which requires to get camera Mac which is used by our application to validate the License. 

We use the command "cat /etc/conf.d/mac" in cameras with old firmware and the camera will return the mac id correctly. This command cat /etc/conf.d/mac is NOT Working in Axis camera with latest firmware. 

Camera model p1375-E with Old firmware "9.80.3.2" - cat /etc/conf.d/mac is working
Camera model p1375-E with New firmware "10.12.104" - cat /etc/conf.d/mac Not working

Could you let us know how to get Mac ID in Axis camera with new firmware usign command.

Thanks,
Rafi

**Comment by rafiomeon** [comment]
We found that cat /sys/class/net/*/address returns the Mac ID of Axis camera with new firmware. We are stuck with another command to get Camera firmware version ... cat < /etc/release | cut -d "-" -f3. This command for firmware version was already working in old firmware. Not sure why Axis is removing commands which was already working.

Could we get command to get firmware version while using New firmware.

Thanks,
Rafi

**Comment by shadysp** [comment, marked as answer]
One of these should return the mac id/address/serial number/hostname `axis-b8xxxxxxxxxx`

- `uname -n` 
  
- `cat /etc/hostname`

- `sysctl kernel.hostname`

- `cat /proc/sys/kernel/hostname`

For the firmware. I will get back to you with an update hopefully soon.

**Reply by shadysp** [reply]
You can get the Firmware version using one of these.

- `grep '^VERSION_ID' /etc/os-release`

- `cat  /etc/os-release`

- `cat /usr/lib/os-release`

- `hostnamectl`
